Resources/Prototypes/_RMC14/Roles/Ranks/Colony/civilan.yml

```
@@ -0,0 +1,4 @@
+﻿- type: rank
+  id: RMCRankCivilan
+  name: Civilan
```

```suggestion
  name: Civilian
```

On mobile so can't find if there's any other instance of this

---

thanks